Platon/backend/tests/issues_test.py

```diff
@@ -52,10 +52,10 @@ def setUp(self):
         db.session.commit()
 
         workspaces = [
-            Workspace(1, 0, "coronovirus study", WorkspaceState.search_for_collaborator.value), # public workspace by Umut on state 1
-            Workspace(2, 1, "SWE difficulties", WorkspaceState.search_for_collaborator.value), # private workspace by Can on state 1
-            Workspace(4, 1, "honeybadger", WorkspaceState.search_for_collaborator.value), # private workspace by Hilal on state 1
-            Workspace(3, 0, "bos", WorkspaceState.search_for_collaborator.value) # public workspace by Alperen on state 1
+            Workspace(creator_id = 1, is_private = 0, title = "coronovirus study", state = WorkspaceState.search_for_collaborator.value), # public workspace by Umut on state 1
```

Comment:
@cakmadam98 If you add description arguments too, then these parts will be compatible with the default constructor and we will have no problems with these. But it's not urgent, we can disable the tests for now.

Reply:
I've added description argument into the tests.